src/util/colour.h

```diff
@@ -32,7 +32,9 @@
 #define TERM_DIM        (16*1)
 #define TERM_BRIGHT     (16*2)
 
+bool term_init(bool tty_);
```

Comment:
This was previously defined in src/runtime/status.h. Perhaps remove the other definition.